### PR TITLE
Fix option ordering and help text

### LINKED ISSUE #14: Parameter order confusion
I missed this in the PR but `image_name` seems to be acceptable [either as a first parameter or a second](https://github.com/osuosl/openstack_taster/blob/master/bin/openstack_taster#L53-L58).

and the [help text](https://github.com/osuosl/openstack_taster/blob/master/bin/openstack_taster#L15-L17) add to the confusion.

It should be:
`./openstack_taster <image_name> {suite_name} [--create-image]`

The `image_name` should be *required*.
The `suite_name` would default to 'all' if not specified.~
The `--create-image` option cannot work without an `image name` so we shouldn't allow it like that. Also, we should probably rename it to `--create-snapshot` to go better with OpenStack terminologies.

### PR DESCRIPTION
**Pull request by Codysseus**
Fixes #14

### CODE REVIEW

File: bin/openstack_taster [hunk 1]
```diff
@@ -12,12 +12,12 @@ settings = {}
 ARGV << '-h' if ARGV.empty?
 
 parser = OptionParser.new do |opts|
-  opts.banner = "'Usage: openstack_taster <test_suite> <image_name> <option>\n" \
-                "   or: openstack_taster <test_suite>\n" \
+  opts.banner = "Usage: openstack_taster <image_name> {suite_name} [--create-snapshot]\n" \
+                "   or: openstack_taster <image_name> [--create-snapshot]\n" \
                 '   or: openstack_taster <option>'
```
Comment (knightsamar) [inline bin/openstack_taster:17]: This won't work anymore, right?

**Review by knightsamar — CHANGES_REQUESTED** [review]

File: bin/openstack_taster (hunk 1 above)
Comment (Codysseus) [reply bin/openstack_taster:17]: This will work for `--help`, but if they attempt to use `--create-snapshot` it will throw an error.

Comment (knightsamar) [reply bin/openstack_taster:17]: Ok, let's keep it here for future purposes.

**Comment by knightsamar** [comment]
Please increment the patch version by 1 and we are good to merge!

**Comment by Codysseus** [comment]
@knightsamar Bumped!

File: bin/openstack_taster [hunk 2]
```diff
@@ -12,12 +12,12 @@ settings = {}
...
   opts.separator('')
   opts.separator('Available Arguments:')
-  opts.on('-c', '--create-image', 'Create image upon test failure.') do
+  opts.on('-c', '--create-snapshot', 'Create snapshot upon test failure.') do
     settings[:create] = true
```
Comment (knightsamar) [inline bin/openstack_taster]: Can we please name this symbol aptly?

Comment (Codysseus) [reply bin/openstack_taster]: What is your suggestion?
Maybe `-s`?

Comment (knightsamar) [reply bin/openstack_taster]: `settings[:create_snapshot]` ?

Comment (Codysseus) [reply bin/openstack_taster]: Oh, that's what you mean. Sorry I wasn't looking at that. Sure thing!

**Comment by Codysseus** [comment]
@knightsamar Ready!

**Review by knightsamar — APPROVED** [review]

**Pull request merged**